installer/bundle/bundle_skel.sh

```diff
@@ -830,13 +830,18 @@ if [ "$installMode" = "I" -o "$installMode" = "U" ]; then
     fi
     curl_installed
     if [ $? -ne 0 ]; then
+        echo "curl was not found, attempting to install curl..."
+        install_extra_package curl
+    fi
+    check_program_in_path curl
```

Comment:
I do not recall an exact case where the "which" check passed but the package was not installed, or vice versa. Therefore, if you think those checks should be removed from the other package installs, you can clean those lines up.